device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,844 @@
+                "direction": "intake",
+                "target_speed": 35,
+                "tolerance": 20,
+                "led_status": "green"
+            }
+        ],
+        "fan_drawers": [
+            {
+                "name": "Fan drawer 1",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "num_fans": 2,
+                "led_status": "green"
+            },
+            {
+                "name": "Fan drawer 2",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "num_fans": 2,
+                "led_status": "green"
+            },
+            {
+                "name": "Fan drawer 3",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "num_fans": 2,
+                "led_status": "green"
+            },
+            {
+                "name": "Fan drawer 4",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "num_fans": 2,
+                "led_status": "green"
+            }
+        ],
+        "psus": [
+            {
+                "name": "psu 1",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "num_fans": 2,
+                "volatge": 2.1,
+                "power": 302.6,
+                "current": 150.6,
+                "powergood_status": "true",
+                "temperature": 30.125,
+                "temperature_high_threshold": 100,
+                "voltage_high_threshold": 12.1,
+                "voltage_low_threshold": 2,
+                "led_status": "green"
+            },
+            {
+                "name": "psu 2",
+                "presence": "true",
+                "model": "FIXME",
+                "serial": "FIXME",
+                "status": "true",
+                "num_fans": 2,
+                "volatge": 2.1,
+                "power": 302.6,
+                "current": 150.6,
+                "powergood_status": "true",
+                "temperature": 30.125,
+                "temperature_high_threshold": 100,
+                "voltage_high_threshold": 12.1,
+                "voltage_low_threshold": 2,
+                "led_status": "green"
+            }
+        ],
+        "thermal_mgr": [
+		"FIXME:need input from Mellanox for required static data"
+	],
+        "watchdog": [
+		"FIXME:need input from Mellanox for required static data"
+	],
+        "eeprom": [
+		"FIXME:need input from Mellanox for required static data"
+	],
+        "thermals": [
+            {
+                "name": "thermal 1",
+                "presence": "true",
```

Comment:
model, serial, temperature should be dropped

Reply:
fixed

Reply:
Since thermals are fixed, maybe we could re-add "model" here. unless the models of the thermals may change mid-production. Need to confirm with Mellanox.